core/internal/gethwrappers/generation/generate_link/wrap_link.go

```diff
@@ -0,0 +1,71 @@
+// package main is a script for generating a geth golang contract wrappers for
+// the LINK token contract.
+//
+//  Usage:
+//
+// With core/internal/gethwrappers as your working directory, run
+//
+//  go run generation/generate_link/wrap.go
+//
+// This will output the generated file to
+// generated/link_token_interface/link_token_interface.go
+
+package main
+
+import (
+	"crypto/sha256"
+	"fmt"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+
+	"github.com/tidwall/gjson"
+
+	"github.com/smartcontractkit/chainlink/core/internal/gethwrappers"
+	"github.com/smartcontractkit/chainlink/core/utils"
+)
+
+func main() {
+	pkgName := "link_token_interface"
+	fmt.Println("Generating", pkgName, "contract wrapper")
+	className := "LinkToken"
+	tmpDir, cleanup := gethwrappers.TempDir(className)
+	defer cleanup()
+	linkDetails, err := ioutil.ReadFile(filepath.Join(
+		gethwrappers.GetProjectRoot(), "evm-test-helpers/src/LinkToken.json"))
+	if err != nil {
+		gethwrappers.Exit("could not read LINK contract details", err)
+	}
+	if fmt.Sprintf("%x", sha256.Sum256(linkDetails)) !=
```

Comment:
`%x` is for hex? I haven't seen that before

Reply:
Yeah, it's nice.